tools/apiview/parsers/swagger-api-parser/global.json

```diff
@@ -0,0 +1,7 @@
+{
+  "sdk": {
```

Comment:
This file should be deleted, per [this teams chat](https://teams.microsoft.com/l/message/19:59dbfadafb5e41c4890e2cd3d74cc7ba@thread.skype/1675891427979?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1675885183657&teamName=Azure%20SDK&channelName=Engineering%20System%20%F0%9F%9B%A0%EF%B8%8F&createdTime=1675891427979).

Reply:
- https://github.com/Azure/azure-sdk-tools/pull/5380